### PR TITLE
SPCTR-1738 Responsive and Display Condition - Added responsive and display condition for core block again.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1738

### Screenshots

- Display and Responsive condition in editor - https://share.getcloudapp.com/P8u7jZzD

### Types of changes

- New

### How has this been tested?

- Tested as per card description.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-init-blocks.php [hunk 1]
```diff
@@ -174,38 +174,12 @@ public function browser_visibility( $block_attributes, $block_content ) {
...
 
-				// Additional check for Chrome that returns Safari.
-				if ( 'safari' === $value || 'firefox' === $value ) {
-					if ( false !== strpos( $_SERVER['HTTP_USER_AGENT'], 'Chrome' ) ) {
-						$show = false;
-					}
-				}
-			}
-		}
+		$show = is_array( $value ) && ! empty( $value ) ? in_array( $user_agent, $value, true ) : $value === $user_agent;
```
Comment (RajkiranBagal) [inline classes/class-uagb-init-blocks.php:182]: Why have we checked array conditions here? In which case does $value comes as array?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: classes/class-uagb-init-blocks.php (hunk 1 above)
Comment (AbhijitNage123) [reply classes/class-uagb-init-blocks.php]: Updated condition and tested.

File: classes/class-uagb-init-blocks.php [hunk 2]
```diff
@@ -174,38 +174,12 @@ public function browser_visibility( $block_attributes, $block_content ) {
...
-		$show = false;
-
-		if ( 'ie' === $value ) {
-			if ( false !== strpos( $_SERVER['HTTP_USER_AGENT'], $browsers[ $value ][0] ) || false !== strpos( $_SERVER['HTTP_USER_AGENT'], $browsers[ $value ][1] ) ) {
-				$show = true;
-			}
-		} else {
-			if ( false !== strpos( $_SERVER['HTTP_USER_AGENT'], $browsers[ $value ] ) ) {
-				$show = true;
+		$show       = false;
```
Comment (RajkiranBagal) [inline classes/class-uagb-init-blocks.php:179]: Why Is this variable definition needed?

File: lib/astra-notices/notices.css [hunk 3]
```diff
@@ -36,17 +36,4 @@
...
-    display: flex;
-    justify-content: right;
-}
-.rtl .astra-notice-container .notice-content {
-    margin-right: 15px;
-}
-.rtl .astra-review-notice-container .dashicons {
-    padding-right: 10px;
-}
+}
```
Comment (RajkiranBagal) [inline lib/astra-notices/notices.css:39]: I think we need this CSS

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: lib/astra-notices/notices.css (hunk 3 above)
Comment (AbhijitNage123) [reply lib/astra-notices/notices.css:39]: I'm not sure how it gets deleted. I'll add it again

File: classes/class-uagb-init-blocks.php (hunk 2 above)
Comment (AbhijitNage123) [reply classes/class-uagb-init-blocks.php]: Removed unwanted $show variable definition from file.

File: lib/astra-notices/notices.css (hunk 3 above)
Comment (AbhijitNage123) [reply lib/astra-notices/notices.css]: Updated rtl CSS astra notice back

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**